packages/websocket-plugin/tests/websocket.module.spec.ts

```diff
@@ -23,7 +23,7 @@ import {
   WebsocketMessageError
 } from '../';
 
-fdescribe('NgxsWebsocketPlugin', () => {
```

Comment:
haha! @arturovt yes, this is the danger of the `fdescribe`! Sometimes you forget to remove it 🤣